scripts/build.sh

```diff
@@ -22,11 +22,12 @@ echo_err() {
 usage() {
 	echo -e "Usage: ${1} [release] [install] [deploy] [assets] [clean]"
 	echo -e "\nCommands:\n"
-	echo -e "  release  builds a release of the current working-tree"
-	echo -e "  install  builds a release and install it the GOPATH"
-	echo -e "  deploy   builds a release of the current working-tree and deploys it"
-	echo -e "  assets   move and download all the required assets (see: ./assets/externals)"
-	echo -e "  clean    remove all generated files from the working-tree"
+	echo -e "  release     builds a release of the current working-tree"
+	echo -e "  install     builds a release and install it the GOPATH"
+	echo -e "  deploy      builds a release of the current working-tree and deploys it"
+	echo -e "  docker-dev  builds a cozy-app-dev docker image"
```

Comment:
Can you update https://github.com/cozy/cozy-stack/blob/master/docs/docker.md#building-the-cozy-app-dev-image with this new command?